crates/aptos-jwk-consensus/src/observation_aggregation/mod.rs

```diff
@@ -57,6 +57,7 @@ impl BroadcastStatus<JWKConsensusMsg> for Arc<ObservationAggregationState> {
             author == sender,
             "adding peer observation failed with mismatched author",
         );
+        ensure!(self.epoch_state.verifier.get_voting_power(&author).is_some(), "adding peer observation failed with illegal signer");
```

Comment:
I "think" this is redundant, the signature verification returns some similar errors